packages/applications-service-api/src/utils/sort-applications.merge.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This is a temporary file which will be removed once the applications-service-api is updated to only use BO applications.
+ * In the meantime while we are merging the applications from BO and NI we need to sort the applications
+ * on the API level
+ */
+
+const sortApplications = (applications, sort = '+ProjectName') => {
+	const direction = sort.startsWith('-') ? 'desc' : 'asc';
+	const key = sort.replace(/^[+-]/, '');
+
+	switch (key) {
+		case 'ProjectName':
+		case 'PromoterName':
+			return applications.sort(sortByStringKeyInObject(key, direction));
+		case 'DateOfDCOSubmission':
+		case 'ConfirmedDateOfDecision':
```

Comment:
What is the behaviour in the absence of this field in the project?  For example if only 2 cases had a ConfirmedDateOfDecision.  

I don't think there is a scenario in the ticket for this but would be good to have a secondary order of ProjectName.

Reply:
Thank you the suggestion - I've added now!